main.py

```diff
@@ -1,10 +1,55 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r", encoding="utf-8") as players_file:
+        players = json.load(players_file)
+
+    for player, info in players.items():
+        nickname = player
+        email = info["email"]
+        bio = info["bio"]
+        race = info["race"]
+        guild = info["guild"]
+
+        if race:
+            skills = race["skills"]
+            player_race, race_created = Race.objects.get_or_create(
+                name=race["name"],
+                description=race["description"],
+            )
+
+        if guild:
+            player_guild, guild_created = Guild.objects.get_or_create(
+                name=guild["name"],
+                description=guild["description"],
+            )
+        else:
+            player_guild = None
+
+        if skills:
```

Comment:
The variable 'skills' is used here but is only defined if 'race' is truthy. This could lead to a NameError if 'race' is falsy. Ensure 'skills' is defined before this line or handle the case where 'race' is falsy.